DependencyInjection/Configuration.php

```diff
@@ -103,6 +103,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('default_format')->defaultNull()->end()
+                        ->scalarNode('method_prefix')->defaultTrue()->end()
```

Comment:
Yeah, I think naming that option `method_name_prefix` was not the best idea. Let's not make the same mistake here again. Can you please rename it? :)